tools/check/ut.go

```diff
@@ -448,6 +450,8 @@ func main() {
 
 	// Get the correct count of CPU if it's in docker.
 	p = runtime.GOMAXPROCS(0)
```

Comment:
Does it work correctly in container environments?

Reply:
Actually, I'm not sure, but as the old code it should be fine.